js/indodax.js

```diff
@@ -995,11 +995,16 @@ module.exports = class indodax extends Exchange {
         const timestamp = this.safeTimestamp2 (transaction, 'success_time', 'submit_time');
         const depositId = this.safeString (transaction, 'deposit_id');
         const feeCost = this.safeNumber (transaction, 'fee');
-        let fee = undefined;
+        let fee = {
```

Comment:
The fee should only be defined if the fee cost is specified.

Reply:
That's inconsistent with our trend of always having keys set to undefined if they don't exist. If `fee` does not have it's sub keys defined then a user in python who tries to access `order['fee']['cost']` will get

```
>>> order['fee']['cost']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not subscriptable
```